src/stories/Button.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { Story, Meta } from '@storybook/html';
```

Comment:
probably can remove these default storybook files?

Reply:
Yes, I think the initial non-USA components are from initializing Storybook.